django_multitenant/tests/test_models.py

```diff
@@ -115,7 +115,7 @@ def test_filter_select_related(self):
             task = Task.objects.filter(pk=task_id).select_related("project").first()
 
             self.assertEqual(task.account_id, task.project.account_id)
-            pattern = 'INNER JOIN "tests_project" ON \("tests_task"."project_id" = "tests_project"."id" AND \("tests_task"."account_id" = .?"tests_project"."account_id"\)\)'
+            pattern = 'INNER JOIN "tests_project" ON \\("tests_task"."project_id" = "tests_project"."id" AND \\("tests_task"."account_id" = .?"tests_project"."account_id"\\)\\)'
```

Comment:
wait, what? how did this pass before?

Reply:
apparently at the moment escaping a character that isn't recognized as an escape sequence results in a literal `\` in front of it. This was marked as deprecated by some semi-recent python versions. Simple test in python REPL:
```python
>>> "\("
'\\('
```